lib/protocol/network.js

```diff
@@ -425,6 +436,7 @@ function cmpAddress(network, prefix) {
     case prefixes.scripthash:
     case prefixes.witnesspubkeyhash:
     case prefixes.witnessscripthash:
+    case prefixes.taproot:
```

Comment:
This fits the design pattern so it makes sense at first look, but it seems weird to have any of the witness prefixes listed here -- the `cmpAddress()` function "should only" be used when parsing a base58 address but I could be wrong:

https://github.com/bcoin-org/bcoin/blob/cf7da94f2fb080706a6c1508c1781bc688cc1f06/lib/protocol/network.js#L311-L313